java/modules/core/src/main/java/org/apache/synapse/util/resolver/CustomJAXPURIResolver.java

```diff
@@ -54,7 +56,10 @@ public CustomJAXPURIResolver(ResourceMap resourceMap, SynapseConfiguration synCf
      * returns null, it will resolve the location using
      * {@link SynapseConfigUtils#resolveRelativeURI(String, String)}.
      */
-    public Source resolve(String href, String base) throws TransformerException {
+    @SuppressWarnings("startswith")//FALSE POSITIVE: method cannot be overriden because of different
+                                   //annotations on params
```

Comment:
Couldn't that be resolved by adding a stub file for the superclass?

Reply:
Or does the property not apply to the superclass? If not, then this might be a bug.